### ISSUE TITLE
computing gradients during backpropagation: How to get gradients of the cross entropy loss w.r.t softmax function in resnet during backpropagation?

### ISSUE THREAD

**Issue by JSP21** [apache/incubator-mxnet#9261]

**Comment by adrianloy** [comment]
If you use the simple_bind api you can get the gradients from the executor object after performing the forward pass.

**Comment by harshitajain1994** [comment]
Suggested Labels : "Question", "HowTo", "API"

**Comment by sandeep-krishnamurthy** [comment]
@JSP21 - Please reopen if your issue still persists.
